src/modules/0.0.22/aws_rds/entity/rds.ts

```diff
@@ -14,15 +15,12 @@ import { AvailabilityZone } from '../../aws_vpc/entity';
 import { ParameterGroup } from './parameter_group';
 
 @Entity()
+@Unique('UQ_identifier_region', ['dbInstanceIdentifier', 'region'])
```

Comment:
It's unlikely that the `dbInstanceIdentifier`s collide despite having multi-region, but I think it's a good idea to keep it unique.

Reply:
So users choose the [`dbInstanceIdentifier` names themselves](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-rds/interfaces/createdbinstancecommandinput.html#dbinstanceidentifier) which means the user accidentally causing a collision is something we should guard against, I think?

What we also need, though, is a check constraint with all of those weird rules on what values are valid. Not sure if it should be in this PR or not, though.

Reply:
Yes. You're right. I thought it's being chosen by the cloud.